src/main/java/io/github/pylonmc/pylon/base/content/machines/simple/VacuumHopper.java

```diff
@@ -74,7 +75,8 @@ public void tick(double deltaSeconds) {
             return; // don't vacuum if powered
         }
 
-        for (Entity entity : getBlock().getLocation().getNearbyEntities(radius + 0.5, radius + 0.5, radius + 0.5)) {
+        Location location = getBlock().getLocation().add(0.5, 0.5, 0.5);
+        for (Entity entity : location.getNearbyEntities(radius + 0.5, radius + 0.5, radius + 0.5)) {
```

Comment:
```suggestion
        for (Entity entity : getBlock().getLocation().toCenterLocation().getNearbyEntities(radius + 0.5, radius + 0.5, radius + 0.5)) {
```
makes intent clearer, and avoids using our own math